ir/src/main/scala/org/scalajs/core/ir/Trees.scala

```diff
@@ -799,6 +799,11 @@ object Trees {
     val tpe = NoType
   }
 
+  case class ClassExportDef(fullName: String)(
+      implicit val pos: Position) extends Tree {
+    val tpe = NoType
+  }
+
```

Comment:
I am not a fan of this name. Is a class export something that will *ever* make sense for a Scala class? IMO it is a good idea to name this `JSClassExportDef` (or something the like). Now it is too easy to read `ModuleExportDef` just next to `ClassExportDef` and think they are equivalent for modules/classes.